warehouse/packaging/views.py

```diff
@@ -288,6 +289,7 @@ def release_detail(release, request):
         "license": license,
         # Additional function to format the attestations
         "PEP740AttestationViewer": PEP740AttestationViewer,
+        "show_share_image": should_show_share_image(request),
```

Comment:
I think it probably makes more sense to pass the user agent here...
```suggestion
        "user_agent": request.user_agent,
```

Reply:
Thanks - I used the request that's already in the template (lmk if that's not always present). I also added some handling for `Request.user_agent is None` in case the user-agent isn't supplied.